students/daniel_grubbs/lesson02/print_grid.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+
+
+def print_grid(n):
+    for num in range(n):
+        print('+', '- ' * 4, end='')
+        print('+', '- ' * 4, end='',)
+        print('+')
+        for i in range(4):
+            print('|', '  ' * 4, end='')
+            print('|', '  ' * 4, end='')
+            print('|')
+    print('+', '- ' * 4, end='')
+    print('+', '- ' * 4, end='',)
+    print('+')
+
+print_grid(3)
+print_grid(15)

```

Comment:
So the goal here was to affect the width of the grid squares directly instead of number of boxes.
Results should look like this for 3 and 5:
```
+-+-+
| | |
+-+-+
| | |
+-+-+
+-------+-------+
|       |       |
|       |       |
|       |       |
|       |       |
|       |       |
|       |       |
|       |       |
+-------+-------+
|       |       |
|       |       |
|       |       |
|       |       |
|       |       |
|       |       |
|       |       |
+-------+-------+```